dist/index.js

```diff
@@ -9569,6 +9575,31 @@ function downloadRepository(authToken, owner, repo, ref, commit, repositoryPath)
     });
 }
 exports.downloadRepository = downloadRepository;
+/**
+ * Looks up the default branch name
+ */
+function getDefaultBranch(authToken, owner, repo) {
+    return __awaiter(this, void 0, void 0, function* () {
+        return yield retryHelper.execute(() => __awaiter(this, void 0, void 0, function* () {
+            core.info('Retrieving the default branch name');
+            const octokit = new github.GitHub(authToken);
+            const response = yield octokit.repos.get({ owner, repo });
```

Comment:
@ericsciple We are using this action on our repository's `.wiki` repo, and our respective action has been failing recently as the API returns "Not Found" for those. Could this be an oversight here?

Reply:
Thanks for the heads up! And sorry for the disruption :(

I will look into the scenario shortly...

Reply:
Thanks! Fortunately was an easy fix on our end :)

Is it even possible to change the wiki default branch name? If not, maybe for those repositories the old hard-coded "logic" could be maintained for now.